src/scenes/home/landing/membership/membership.css

```diff
@@ -8,13 +8,12 @@
   display: flex;
   flex-flow: row wrap;
   justify-content: space-around;
-  width: 80%;
+  width: 90%;
```

Comment:
Go ahead and just remove this line. The parent Section component already limits the width of all children components.